packages/block-editor/src/components/default-style-picker/index.js

```diff
@@ -3,56 +3,88 @@
  */
 import { store as blocksStore } from '@wordpress/blocks';
 import { useMemo, useCallback } from '@wordpress/element';
-import { __ } from '@wordpress/i18n';
-import { SelectControl } from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
 import { useSelect } from '@wordpress/data';
 
 /**
  * Internal dependencies
  */
 import { store as blockEditorStore } from '../../store';
+import { getActiveStyle } from '../block-styles/utils';
 
-export default function DefaultStylePicker( { blockName } ) {
+export default function DefaultStylePicker( { clientId } ) {
```

Comment:
Is this change needed?

Reply:
I need to get the block attributes to work out the active style, so I make use of `getBlock( clientId )`. 

I can remove that argument altogether and use `getSelectedBlock()` in the `useSelect`, which I'm doing locally now and it works as well.

Is there a more optimal/Gutenbergy way?